Week1/homework/5-Round a number and log it.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const z = 7.25;
+console.log(z);
```

Comment:
You did not round the number. Please carefully read the instructions and improve your solution.

Reply:
ohhh sorry I corrected it but not saved I will do that again